housekeeping-vacuum-tool/README.md

```diff
@@ -1,34 +1,74 @@
-## Housekeeping Vacuum Tool
+# Housekeeping Vacuum Tool
 
-### Usage
+# Overview
 
-#### YAML Configuration
+The Housekeeping Vacuum Tool looks for any files and folders in the data locations of a Hive table that are not referenced in either the Hive metastore or the Housekeeping database. Any paths discovered are considered "orphaned data" and are then scheduled for removal via the Housekeeping process. This can be useful if a Housekeeping database has become corrupt and previous orphaned path locations have been lost. By running the Vacuum Tool against a Hive table one can identify data that is no longer referenced by Hive and is thus a candidate for deletion.  
+
+## Install
+
+### Pre-requisites
```

Comment:
Well, you don't need a cluster, you just need a machine that has the Hadoop and Hive clients and libraries on it, which is what I was trying to say below. Perhaps I should mention the CLI client apps specifically?

Reply:
I said that because it's got a specific section on how to run this on EMR. Perhaps making a note saying it does not require a Hadoop cluster, but requires the Hadoop libraries to be present in the classpath is more than enough.

My first comment might be a nice note for the EMR section.